### PR TITLE
fix codestyle issues

### PR DESCRIPTION
**Pull request by Nolfgk**
fix codestyle issues
error handling

### CODE REVIEW

File: server/authMiddleware.go [hunk 1]
```diff
@@ -95,7 +96,10 @@ func respondWithErrorString(w http.ResponseWriter, code int, message string) {
 
 //respondWithErrorString converts message JSON ans sends 401 to the client
 func respondWithError(w http.ResponseWriter, code int, message interface{}) {
-	WriteJSON(code, message, w)
+	e := WriteJSON(code, message, w)
+	if nil != e {
+		log.Print(e)
```
Comment (avarabyeu) [inline server/authMiddleware.go]: please, use Logrus and Error level

File: server/authMiddleware.go [hunk 2]
```diff
@@ -126,19 +130,29 @@ func getTokenInfo(token string, authServerURL string) (*User, error) {
 	if nil != e {
 		return nil, e
 	}
-	defer rs.Body.Close()
+	defer func() {
+		if err := rs.Body.Close(); err != nil {
+			log.Print(err)
```
Comment (avarabyeu) [inline server/authMiddleware.go]: please, use Logrus and Error level

File: server/authMiddleware.go [hunk 3]
```diff
@@ -126,19 +130,29 @@ func getTokenInfo(token string, authServerURL string) (*User, error) {
...
 			errorDesc:  uatErr,
 			statusCode: rs.StatusCode,
 		}
 		return nil, e
 	}
 	user := new(User)
-	decodeJSON(rs, user)
+	err := decodeJSON(rs, user)
+	if nil != err {
+		return nil, err
```
Comment (avarabyeu) [inline server/authMiddleware.go:160]: please, use Logrus and Error level

File: server/rpserver.go [hunk 4]
```diff
@@ -98,11 +98,17 @@ func (srv *RpServer) initDefaultRoutes() {
...
+			log.Print(e)
+		}
 	})
 
 	bi := map[string]interface{}{"build": srv.buildInfo}
 	srv.mux.Get("/info", func(w http.ResponseWriter, rq *http.Request) {
-		WriteJSON(http.StatusOK, bi, w)
+		e := WriteJSON(http.StatusOK, bi, w)
+		if nil != e {
+			log.Print(e)
```
Comment (avarabyeu) [inline server/rpserver.go]: please, use Logrus and Error level

File: server/rpserver.go [hunk 5]
```diff
@@ -98,11 +98,17 @@ func (srv *RpServer) initDefaultRoutes() {
 			rs["status"] = "UP"
 		}
 
-		WriteJSON(status, rs, w)
+		e := WriteJSON(status, rs, w)
+		if nil != e {
+			log.Print(e)
```
Comment (avarabyeu) [inline server/rpserver.go]: please, use Logrus and Error level

**Review by avarabyeu — CHANGES_REQUESTED** [review]

File: server/rpserver.go (hunk 5 above)
Comment (DzmitryHumianiuk) [reply server/rpserver.go]: @Nolfgk

File: server/rpserver.go (hunk 4 above)
Comment (DzmitryHumianiuk) [reply server/rpserver.go]: @Nolfgk

File: server/authMiddleware.go (hunk 3 above)
Comment (DzmitryHumianiuk) [reply server/authMiddleware.go:160]: @Nolfgk

File: server/authMiddleware.go (hunk 2 above)
Comment (DzmitryHumianiuk) [reply server/authMiddleware.go]: @Nolfgk

File: server/authMiddleware.go (hunk 1 above)
Comment (DzmitryHumianiuk) [reply server/authMiddleware.go]: @Nolfgk

Comment (Nolfgk) [reply server/authMiddleware.go]: done

File: server/authMiddleware.go (hunk 3 above)
Comment (Nolfgk) [reply server/authMiddleware.go:160]: @avarabyeu  not sure if i used it correctly, please check (i added log.Error here).

File: server/rpserver.go (hunk 4 above)
Comment (Nolfgk) [reply server/rpserver.go]: done

File: server/authMiddleware.go (hunk 2 above)
Comment (Nolfgk) [reply server/authMiddleware.go]: done

**Comment by DzmitryHumianiuk** [comment]
@avarabyeu you are only one, who can review it

**Comment by avarabyeu** [comment]
guys, all the changes are applied as part of separate PR. @Nolfgk thanks for the contribution!

**Pull request closed without merging**